src/main/resources/db/changelog/20180207_1707.mysql.sql

```diff
@@ -0,0 +1,10 @@
+--liquibase formatted sql
+
+--changeset chris:1518019686854-1
+ALTER TABLE facility ADD facility_id VARCHAR(255) NOT NULL;
+
+--changeset chris:1518019686854-2
+ALTER TABLE facility ADD CONSTRAINT UC_FACILITYFACILITY_ID_COL UNIQUE (facility_id);
+
+--changeset chris:1518019686854-3
+ALTER TABLE facility MODIFY type INT NOT NULL;
```

Comment:
we don't store enums numbers in the database, so you will need to update this after adding the annotation